### PR TITLE
Reorganize cn_metrics.py output

### PR DESCRIPTION
**Pull request by beckerg**
Signed-off-by: Greg Becker <gbecker@micron.com>

### CODE REVIEW

**Comment by beckerg** [comment]
Beware, I have no python chops..

File: tools/scripts/cn_metrics.py [hunk 1]
```diff
@@ -43,60 +43,64 @@
...
 
-def full_tree(ybuf: Optional[Dict[str, Any]], opt):
+def full_tree(ybuf: Optional[OrderedDict[str, Any]], opt):
     if not ybuf or "info" not in ybuf:
         return
 
-    print("t ", end=""),
-    oids = ["oid1", "oid2"]
+    # Vector of initial column widths (i.e., compc, dgen, keys, ...)
+    #
```
Comment (tristan957) [inline tools/scripts/cn_metrics.py:51]: Drop extra comment line

File: tools/scripts/cn_metrics.py [hunk 2]
```diff
@@ -43,60 +43,64 @@
...
             continue
 
-        if node["info"]["nkvsets"] == 0:
+        if node["info"]["kvsets"] == 0:
             print()
             continue
 
-        # print info for all kvsets in current node
+        # Print info for each kvset in the current node.
+        #
```
Comment (tristan957) [inline tools/scripts/cn_metrics.py:94]: Drop extra comment line.

**Comment by tristan957** [comment]
Do you have a comparison of what the outputs looked like before and after?

File: tools/scripts/cn_metrics.py (hunk 1 above)
Comment (beckerg) [reply tools/scripts/cn_metrics.py:51]: The general idea is to let the code breathe, bind the comment to the code following it, but the intervening blank line makes the code much easier to read versus when it's directly preceded by a comment.

File: tests/unit/kvdb/kvdb_rest_test.c [hunk 3]
```diff
@@ -407,58 +411,58 @@ MTF_DEFINE_UTEST_PREPOST(kvdb_rest, print_tree_test, test_pre, test_post)
...
+        "    tombs: 0\n"
+        "    ptombs: 0\n"
+        "    hlen: 2000000\n"
+        "    klen: 4000000\n"
+        "    vlen: 8000000\n"
+        "    hblks: 1\n"
+        "    kblks: 1\n"
+        "    vblks: 1\n"
+        "    vgroups: 1\n"
+        "    hblkids: 0x70310c\n"
```
[the remark this answers is not in the document]
Comment (beckerg) [reply tests/unit/kvdb/kvdb_rest_test.c:433]: Done.

File: tools/scripts/cn_metrics.py (hunk 1 above)
Comment (tristan957) [reply tools/scripts/cn_metrics.py:51]: I don't see it the same way as you, but sounds good...

**Comment by beckerg** [comment]
I'll update the ticket with before/after results.

**Review by alexttx — APPROVED** [review]

**Review by nabeelmmd — APPROVED** [review]

**Comment by beckerg** [comment]
Thanks guys!  Who knew changing the output of cn_metrics would break a smoke test?

**Pull request merged**